### PR TITLE
7.0 hr_contract_hourly_rate add constraint on wage

### PR DESCRIPTION
**Pull request by ghost**
Remove the required attribute on wage. Instead, add a constraint that checks the salary computation method is 'wage' and the wage is not filled.

### CODE REVIEW

File: hr_contract_hourly_rate/hr_contract.py [hunk 1]
```diff
@@ -99,11 +103,35 @@ class assigned to it.
...
+    ):
+        """
+        Check if the wage on the contract is not False
+        if the salary computation method is wage
+        """
+        for contract in self.browse(cr, uid, ids, context=context):
+
+            # This does not apply when employee is paid by hourly rate
+            if contract.salary_computation_method == 'wage':
+                if contract.wage is False:
```
Comment (guewen) [inline hr_contract_hourly_rate/hr_contract.py]: `if not contract.wage:` would be better IMO

**Comment by guewen** [comment]
I added one remark, apart that LGTM
Thanks

File: hr_contract_hourly_rate/hr_contract.py (hunk 1 above)
Comment (ghost) [inline hr_contract_hourly_rate/hr_contract.py]: I agree, Thanks!

**Comment by max3903** [comment]
:+1:

**Pull request merged**